tools/its.py

```diff
@@ -794,23 +794,23 @@ def terminpaar_im_zeitrahmen(terminpaar, zeitrahmen):
     :param zeitrahmen: Zeitrahmen-Dictionary wie in ImpfterminService.termin_suchen
     :return: True oder False
     """
-    if zeitrahmen is None:
+    if not zeitrahmen: # Teste auf leeres dict
         return True
 
     assert zeitrahmen["einhalten_bei"] in ["1", "2", "beide"]
 
-    von_datum = datetime.datetime.strptime(
+    von_datum = datetime.strptime(
```

Comment:
Hier war das Problem, dass in dieser Datei (anders als in `tools/kontaktdaten.py`) direkt `datetime.datetime` importiert wird. Analog `date` und `dtime`.